### PR TITLE
Refactor sfr lauren

### PR DESCRIPTION
**Pull request by ljgreenhill**

### CODE REVIEW

File: src/SFRField.hpp [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    }
+
+    operator T()
+    {
+        return value;
+    }
+    T get() { return value; }
+    T set(T input)
+    {
+        if (bounded && input <= max && input >= min) {
```
Comment (asmith22) [inline src/SFRField.hpp:69]: This clause is partially inverted. We want to set if its is not bounded or if it is bounded and within the range
if(!bounded || 
```suggestion
        if (bounded && input <= max && input >= min) {
```

File: src/sfr.hpp [hunk 2]
```diff
@@ -38,13 +39,15 @@
 
 namespace sfr {
     namespace stabilization {
-        extern float max_time;
-    }
+        // SFRField<float> max_time(1.0f, 0.0f, 2.0f, 1001);
```
Comment (asmith22) [inline src/sfr.hpp:42]: Normally would req removing commented code but its mine so I'll remove it in the next PR

**Review by asmith22 — CHANGES_REQUESTED** [review]
Single line fix plus questions

File: test/test_command_monitor/test_command_monitor.cpp [hunk 3]
```diff
@@ -405,12 +405,12 @@ void test_down_period()
 
     // downlink period 2 hours
     uint16_t f_opcode = command_monitor.get_decimal_opcode(constants::rockblock::rockblock_downlink_period);
-    uint32_t f_arg_1 = constants::time::two_hours;
+    uint32_t f_arg_1 = 2 * constants::time::one_hour;
     RockblockCommand command = RockblockCommand(f_opcode, f_arg_1, 0);
     sfr::rockblock::processed_commands.push_back(command);
     sfr::rockblock::waiting_command = true;
```
[the remark this answers is not in the document]
Comment (ljgreenhill) [reply test/test_command_monitor/test_command_monitor.cpp:411]: Are you saying just check if the length of the queue has changed?

[the remark this answers is not in the document]
Comment (ljgreenhill) [reply test/test_command_monitor/test_command_monitor.cpp:411]: Merging for now- I think this is a separate issue that could be addressed later

**Pull request merged**